.circleci/config.yml

```diff
@@ -244,12 +244,16 @@ jobs:
       inventory:
         type: string
         default: "ci.ini"
+      apm_enabled:
+        type: string
+      remote_updates:
+        type: string
     docker:
       - image: datadog/docker-library:ansible_<<parameters.os>>_<<parameters.ansible_version>>
     steps:
       - checkout
-      - run: ANSIBLE_JINJA2_NATIVE="<<parameters.jinja2_native>>" ansible-playbook -i ./ci_test/inventory/<<parameters.inventory>> "./ci_test/install_installer.yaml"
-      - run: datadog-installer version
+      - run: ANSIBLE_JINJA2_NATIVE="<<parameters.jinja2_native>>" ansible-playbook -i ./ci_test/inventory/<<parameters.inventory>> "./ci_test/install_installer.yaml" -e datadog_apm_instrumentation_enabled="<<parameters.apm_enabled>>" -e datadog_remote_updates="<<parameters.remote_updates>>"
+      - run: bash -c 'if [ -n "<<parameters.apm_enabled>>" ] || [ "<<parameters.remote_updates>>" = "true" ]; then datadog-installer version; elif command -v datadog-installer; then echo datadog-installer should not be installed; exit 2; else echo datadog-installer is not installed as expected; fi'
```

Comment:
nit: I am not sure we need `datadog-installer is not installed as expected` and I am a bit curious about whether it's possible to have smaller lines

Reply:
agreed on the line size, this is handled in a follow up PR: https://github.com/DataDog/ansible-datadog/pull/589/commits/3a0780379b9fc0e1bccac4cd5f552f05fecf2c56
Although as it's currently kind of blocked, I'll cherry pick it here

Regarding the `echo ... as expected` this was to avoid having a large chunk of shell script being displayed with no insight, I find it a bit reassuring to have the test job saying what failed/succeeded